GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationView.swift

```diff
@@ -0,0 +1,171 @@
+import Foundation
+import UIKit
+
+extension SecureConversations {
+    final class ConfirmationView: View {
+        static let sideMargin = 24.0
+        static let confirmationImageSize = 144.0
+
+        struct Props: Equatable {
+            let style: ConfirmationStyle
+            let backButtonTap: Cmd
+            let closeButtonTap: Cmd
+            let checkMessageButtonTap: Cmd
+        }
+
+        let header: Header
```

Comment:
Can `header` be `private` property?

Reply:
Yep, fixed a16605c00a020a414d02c779c490b34f9cd9e93a